pkg/mcs/tso/server/server.go

```diff
@@ -411,6 +404,28 @@ func (s *Server) SetExternalTS(externalTS uint64) error {
 	return nil
 }
 
+// ResetTS resets the TSO with the specified one.
+func (s *Server) ResetTS(ts uint64, ignoreSmaller, skipUpperBoundCheck bool, keyspaceGroupID uint32) error {
+	log.Info("reset-ts",
```

Comment:
Will this log be overwhelmed?

Reply:
It is the same as the PD mode. I think it's OK?